mono/eglib/goutput.c

```diff
@@ -57,19 +57,23 @@ g_assert_abort (void)
 		abort ();
 }
 
-void
+gint
```

Comment:
Can we start using plain `int`? Notice `ret`'s type..

Reply:
Yeah I should have used that, though I don't think changing it over merits another round of CI.